src/components/Experiences/index.jsx

```diff
@@ -0,0 +1,273 @@
+import React from 'react';
+import { Container } from '../Container';
+import { Section, Stage, StagesContainer, Title, StageCard, StageImage, StageName, StageLocation, StageHover, StageButton } from './styles';
+import ang from '../../images/experiences23/ang.jpg';
+import cc from '../../images/experiences23/cc.jpg';
+import koa from '../../images/experiences23/KOA.png';
+import morph from '../../images/experiences23/morph.png';
+import ms from '../../images/experiences23/ms.png';
+import sc from '../../images/experiences23/sc.jpg';
+import ss from '../../images/experiences23/ss.png';
+import tc from '../../images/experiences23/tc.png';
+import zg from '../../images/experiences23/zg.jpg';
+import tpoko from '../../images/experiences23/tpoko.jpg';
+import metadyne from '../../images/experiences23/metadyne.jpg';
+import smiley from '../../images/experiences23/smiley.jpg';
+import mf from '../../images/experiences23/mf.jpg';
+import gn from '../../images/experiences23/GAIAN.jpg';
+import dwow from '../../images/experiences23/dwow.png';
+
+
+const Experiences = () => (
+    <Section id="experiences">
+        <Container>
+            <Title>
+                E<span style={{'font-family': 'SaintRegular'}}>X</span>PERIENCES
+            </Title>
+            <StagesContainer>
+                <Stage href="https://events.decentraland.org/event/?id=06ed733b-8a0d-48e4-b0c3-17c1ab0e3dac" target="_blank">
```

Comment:
Would you mind using the correct link for the site?
```suggestion
                <Stage href="https://decentraland.org/events/event/?id=06ed733b-8a0d-48e4-b0c3-17c1ab0e3dac" target="_blank">
```
Would you mind to also add the `rel="noopener"` attribute to all the anchors?